cmd/placement/options/options.go

```diff
@@ -148,3 +177,22 @@ func parsePeersFromFlag(val []string) []raft.PeerInfo {
 
 	return peers[:i]
 }
+
+// getEnvIntWithRange returns the integer value of the environment variable with the given name.
+func getEnvIntWithRange(envVar string, defaultValue int, min int, max int) (int, error) {
```

Comment:
Cut a PR in dapr/kit too. I will not block this, but it can be an easy follow up to remove this utility method from here.

Reply:
I expected this feedback :) 
Ok, on it.

Reply:
Here's the kit PR: https://github.com/dapr/kit/pull/102